cosmovisor/scanner_test.go

```diff
@@ -1,63 +1,50 @@
-package cosmovisor_test
+package cosmovisor
```

Comment:
```suggestion
package cosmovisor_test
```

Reply:
I don't need to do external tests. I'm testing only the current package.